src/devices/Vl53L0X/Vl53L0X.cs

```diff
@@ -0,0 +1,1114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This code has been ported from the Dexter Industries Python code
+// https://github.com/DexterInd/DI_Sensors/blob/master/Python/di_sensors/VL53L0X.py
+// It is based as well on the offical ST Microelectronics API in C
+// https://www.st.com/content/st_com/en/products/embedded-software/proximity-sensors-software/stsw-img005.html
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+namespace Iot.Device.Vl53L0X
+{
+    public class Vl53L0X : IDisposable
+    {
+        /// <summary>
+        /// The default I2C Address, page 18 of the documentation 0x52 >> 1 = 0x29
```

Comment:
>  page 18 of the documentation 0x52 >> 1 = 0x29

I don't think this sentence should be put in `<summary>`.

Reply:
ok, I'll move it to comments. Dépends of the PR and people commenting, sometimes, it is asked to be there, sometimes not. So we should aligne :-)